styles/gauntlet.css

```diff
@@ -63,6 +69,22 @@ SELECT LAYOUT STYLES
 	margin-bottom: 20px;
 }
 
+#species-select[disabled],
+#class-select[disabled],
+#weapon-select[disabled],
+input[disabled] {
+	background-color: lightgrey;
+}
```

Comment:
I can't replicate this, and the background color shouldn't affect it. Once everything is merged, I will see if it happens and go from there.